docs/testing-components/test-driven-development/index.md

```diff
@@ -88,37 +85,35 @@ solution.
-*   `setProps()` - Manually sets the props of the component
+```js
+const {...Results} = render(<Component {...props} />, {...Options});
+```
 
-### Full Rendering - `mount()`
+When we render the component we have a number of render options:
 
-Full rendering or [mount](https://github.com/enzymejs/enzyme/blob/master/docs/api/mount.md) renders the component specified as well as all children.  This can be useful when you need to be able to
-test that properties get passed to children. Mount rendering uses the same methods as Shallow rendering listed above.
+* `{container}` - By default, React Testing Library will create a div and append that div to the document.body and this is where your React component will be rendered. If you provide your own HTMLElement container via this option, it will not be appended to the document.body automatically
+* `{baseElement}` - If the container is specified, then this defaults to that, otherwise this defaults to document.body. This is used as the base element for the queries as well as what is printed when you use debug()
+* `{hydrate}` - If hydrate is set to true, then it will render with ReactDOM.hydrate. This may be useful if you are using server-side rendering and use ReactDOM.hydrate to mount your components.
+* `{legacyRoot}` - Used for apps that requires rendering like in React 17 or older
+* `{wrapper}` - Pass a React Component as the wrapper option to have it rendered around the inner elementReturns the props of the component
+* `{queries}` - Queries to bind. Overrides the default set from DOM Testing Library unless merged.
 
-### Static Rendering - `render()`
+There are also some render results:
 
-Static rendering or [render](https://github.com/enzymejs/enzyme/blob/master/docs/api/render.md), generates the HTML output from the specified component.  Static rendering has several utility methods including:
+* `{queries}` -  The most important feature of render is that the queries from DOM Testing Library are automatically returned with their first argument bound to the baseElement, which defaults to document.body. See [Queries](https://testing-library.com/docs/queries/about/) for a complete list.
+* `{container}` - The containing DOM node of your rendered React Element (rendered using ReactDOM.render)
+* `{baseElement}` - If the container is specified, then this defaults to that, otherwise this defaults to document.body. This is used as the base element for the queries as well as what is printed when you use debug()
+* `{hydrate}` - If hydrate is set to true, then it will render with ReactDOM.hydrate. This may be useful if you are using server-side rendering and use ReactDOM.hydrate to mount your components
+* `{debug}` - This method is a shortcut for console.log(prettyDOM(baseElement))
+* `{rerender}` - This function can be used to update props of the rendered component
+* `{unmount}` - This will cause the rendered component to be unmounted
```

Comment:
```suggestion
* `{debug}` - This method is a shortcut for console.log(prettyDOM(baseElement)).
* `{rerender}` - This function can be used to update props of the rendered component.
* `{unmount}` - This will cause the rendered component to be unmounted.
```